charts/consul/templates/connect-inject-deployment.yaml

```diff
@@ -41,9 +42,25 @@ spec:
         {{- if (and .Values.global.secretsBackend.vault.enabled .Values.global.tls.enabled) }}
         "vault.hashicorp.com/agent-init-first": "true"
         "vault.hashicorp.com/agent-inject": "true"
+        {{- if .Values.global.secretsBackend.vault.consulConnectInjectCARole }}
```

Comment:
we have a separate role because someone can choose to mount PKI for connect injector webhooks at different path than the connect CA PKI

Reply:
Done.  Thank you.  I also renamed `consulControllerCARole` to `consulControllerRole`.